docs/api/graphql/token-management.md

```diff
@@ -0,0 +1,123 @@
+# Access Token Management
```

Comment:
Once we add these to the UI we can even point to this from the UI. 

I think we probably need to continue to extend this doc a bit to discuss V1 and V2 tokens. But I'm okay to hold off until first drop